test/UnitTest/Microsoft.AspNet.OData.Test.Shared/OpenComplexTypeTest.cs

```diff
@@ -1,39 +1,27 @@
 ﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
 // Licensed under the MIT License.  See License.txt in the project root for license information.
 
-#if NETCORE
-using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Linq;
-using System.Net;
-using System.Net.Http;
-using System.Net.Http.Headers;
-using System.Threading.Tasks;
-using Microsoft.AspNet.OData.Builder;
-using Microsoft.AspNet.OData.Extensions;
-using Microsoft.AspNet.OData.Test.Abstraction;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.OData.Edm;
-using Newtonsoft.Json.Linq;
-using Xunit;
-#else
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+#if !NETCORE
 using System.Web.Http;
+#endif
 using Microsoft.AspNet.OData.Builder;
 using Microsoft.AspNet.OData.Extensions;
 using Microsoft.AspNet.OData.Test.Abstraction;
+#if NETCORE
+using Microsoft.AspNetCore.Mvc;
+#endif
 using Microsoft.OData.Edm;
 using Newtonsoft.Json.Linq;
 using Xunit;
-#endif
```

Comment:
keep the: 

#if NETCORe

#else

#endif

don't nest multiple #if #endif

Reply:
OK. I thought we can save some duplicated lines of using if they are used by both.
I am OK with either way.

---
In reply to: [190751449](https://github.com/OData/WebApi/pull/1445#discussion_r190751449) [](ancestors = 190751449)